cloudsql/sqlserver/database-sql/Dockerfile

```diff
@@ -43,6 +43,12 @@ RUN set -x && apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -
 # Copy the binary to the production image from the builder stage.
 COPY --from=builder /app/server /app/server
 COPY ./templates /app/templates
+# Copy any certificates if present.
+COPY ./certs /app/certs
+
+# Use server certificate for encrypted connection.
+COPY ./certs/server-ca.pem /usr/local/share/ca-certificates/server-ca.crt
+RUN update-ca-certificates
```

Comment:
Is this step necessary?

Reply:
Yes. This is how the server certificate is mounted within the container and used to match the certificate provided by the sample app when establishing a connection. Without this command, the sample app fails to connect with the error: "...TLS Handshake failed: x509: certificate signed by unknown authority".